### PR TITLE
Invert block-webaudio default

### LINKED ISSUE #24311: Outdated documentation of media.autoplay.block-webaudio
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/Media/Autoplay_guide

### What specific section or headline is this issue about?

https://developer.mozilla.org/en-US/docs/Web/Media/Autoplay_guide#firefox
https://developer.mozilla.org/en-US/docs/Web/Media/Autoplay_guide#browser_configuration_options

### What information was incorrect, unhelpful, or incomplete?

> media.autoplay.block-webaudio
>
>   A Boolean preference which indicates whether to apply autoplay blocking to the [Web Audio API](https://developer.mozilla.org/en-US/docs/Web/API/Web_Audio_API). The default is false, except on Nightly where it is true.

### What did you expect to see?

It should be documented, that autoplay of videos with sound is disabled by default now. `media.autoplay.block-webaudio` is set to `true` now.

### Do you have any supporting links, references, or citations?

https://github.com/mdn/content/issues/6736
https://support.mozilla.org/en-US/kb/block-autoplay
https://bugzilla.mozilla.org/show_bug.cgi?id=1773577

### Do you have anything more you want to share?

_No response_

### PR DESCRIPTION
**Pull request by SiddheshBangar**
Updated the documentation of media

Updating the content of 'autoplay.block-webaudio' function in the documentation.

So that the documentation might stay up to date and users should be able to know about the updated usage of the function

### Related issues and pull requests

Fixes #24311

### CODE REVIEW

File: files/en-us/web/media/autoplay_guide/index.md [hunk 1]
```diff
@@ -341,7 +341,7 @@ Browsers may have preferences that control the way autoplay works, or how autopl
 - `media.autoplay.allow-muted`
   - : A Boolean preference which if `true` (the default) allows audio media which is currently muted to be automatically played. If this has been changed to `false`, media with an audio track will not be permitted to play even if muted.
 - `media.autoplay.block-webaudio`
-  - : A Boolean preference which indicates whether to apply autoplay blocking to the [Web Audio API](/en-US/docs/Web/API/Web_Audio_API). The default is `false`, except on Nightly where it is `true`.
+  - : A Boolean preference which indicates whether to apply autoplay blocking to the [Web Audio API](/en-US/docs/Web/API/Web_Audio_API). The default is set to `true`, in which the autoplay of the sound is disabled.
```
Comment (hamishwillee) [inline files/en-us/web/media/autoplay_guide/index.md:344]:
```suggestion
  - : A Boolean preference that indicates whether to apply autoplay blocking to the [Web Audio API](/en-US/docs/Web/API/Web_Audio_API).
     The default is set to `true` (autoplay of media with sound is disabled).
```

**Review by hamishwillee — APPROVED** [review]
Excellent - thanks

File: files/en-us/web/media/autoplay_guide/index.md [hunk 2]
```diff
@@ -341,7 +341,8 @@ Browsers may have preferences that control the way autoplay works, or how autopl
 - `media.autoplay.allow-muted`
   - : A Boolean preference which if `true` (the default) allows audio media which is currently muted to be automatically played. If this has been changed to `false`, media with an audio track will not be permitted to play even if muted.
 - `media.autoplay.block-webaudio`
-  - : A Boolean preference which indicates whether to apply autoplay blocking to the [Web Audio API](/en-US/docs/Web/API/Web_Audio_API). The default is `false`, except on Nightly where it is `true`.
+  - : A Boolean preference that indicates whether to apply autoplay blocking to the [Web Audio API](/en-US/docs/Web/API/Web_Audio_API).
+     The default is set to `true` (autoplay of media with sound is disabled).
```
Comment (hamishwillee) [inline files/en-us/web/media/autoplay_guide/index.md:345]: Actually, that "autoplay of media with sound is disabled is incorrect/incomplete". 
```suggestion
     The default is set to `true`.
```

**Review by hamishwillee — CHANGES_REQUESTED** [review]
Actually, putting on hold. The text is accurate, but I want to be sure of the nuance for "by default") is captured: https://bugzilla.mozilla.org/show_bug.cgi?id=1773577

**Comment by SiddheshBangar** [comment]
@hamishwillee I didn't understand? What exact changes do I have to do?

File: files/en-us/web/media/autoplay_guide/index.md [hunk 3]
```diff
@@ -341,7 +341,8 @@ Browsers may have preferences that control the way autoplay works, or how autopl
 - `media.autoplay.allow-muted`
   - : A Boolean preference which if `true` (the default) allows audio media which is currently muted to be automatically played. If this has been changed to `false`, media with an audio track will not be permitted to play even if muted.
 - `media.autoplay.block-webaudio`
-  - : A Boolean preference which indicates whether to apply autoplay blocking to the [Web Audio API](/en-US/docs/Web/API/Web_Audio_API). The default is `false`, except on Nightly where it is `true`.
+  - : A Boolean preference that indicates whether to apply autoplay blocking to the [Web Audio API](/en-US/docs/Web/API/Web_Audio_API).
+     The default is set to `true`.
```
Comment (hamishwillee) [inline files/en-us/web/media/autoplay_guide/index.md:345]: I think we need to be clear what allows autoplay in this case - waiting on response in https://bugzilla.mozilla.org/show_bug.cgi?id=1773577#c6

```suggestion
  - : A Boolean preference that indicates whether to apply autoplay blocking to the [Web Audio API](/en-US/docs/Web/API/Web_Audio_API).
     When this is set, audio contexts are only able to play on pages once there has been {{Glossary("Sticky activation")}}.
     The default is set to `true`.
```

**Comment by hamishwillee** [comment]
@SiddheshBangar YOu don't have to do anything. I think we need to clarify when contexts are allowed to play - I've done that in https://github.com/mdn/content/pull/24314/files#r1111394395 but I want to get confirmation from one of the developers before I apply that and merge.

File: files/en-us/web/media/autoplay_guide/index.md [hunk 4]
```diff
@@ -341,7 +341,10 @@ Browsers may have preferences that control the way autoplay works, or how autopl
 - `media.autoplay.allow-muted`
   - : A Boolean preference which if `true` (the default) allows audio media which is currently muted to be automatically played. If this has been changed to `false`, media with an audio track will not be permitted to play even if muted.
 - `media.autoplay.block-webaudio`
-  - : A Boolean preference which indicates whether to apply autoplay blocking to the [Web Audio API](/en-US/docs/Web/API/Web_Audio_API). The default is `false`, except on Nightly where it is `true`.
+  - : A Boolean preference that indicates whether to apply autoplay blocking to the [Web Audio API](/en-US/docs/Web/API/Web_Audio_API).
+     If false, web audio is always allowed to autoplay.
+     When true, audio contexts are only able to play on pages once there has been {{Glossary("Sticky activation")}}.
+     The default is set to `true`.
```
Comment (hamishwillee) [inline files/en-us/web/media/autoplay_guide/index.md:347]:
```suggestion
  - : A Boolean preference that indicates whether to apply autoplay blocking to the [Web Audio API](/en-US/docs/Web/API/Web_Audio_API).
     If `false`, web audio is always allowed to autoplay.
     If `true`, audio contexts are only able to play on pages once there has been {{Glossary("Sticky activation")}}.
     The default is set to `true`.
```

**Review by hamishwillee — APPROVED** [review]
Thanks very much for your contribution @SiddheshBangar - good to have this up to date.

**Comment by SiddheshBangar** [comment]
Welcome @hamishwillee

**Pull request merged**